birdhouse/config/jupyterhub/jupyterhub_config.py.template

```diff
@@ -19,6 +19,11 @@ c.MagpieAuthenticator.magpie_url = "http://magpie:2001"
 c.MagpieAuthenticator.public_fqdn = "${PAVICS_FQDN_PUBLIC}"
 c.MagpieAuthenticator.authorization_url = "${JUPYTERHUB_AUTHENTICATOR_AUTHORIZATION_URL}"
 
+if os.getenv("JUPYTERHUB_CRYPT_KEY"):
+    c.MagpieAuthenticator.enable_auth_state = True
+    c.MagpieAuthenticator.refresh_pre_spawn = True
+    c.MagpieAuthenticator.auth_refresh_age = int("${JUPYTERHUB_AUTHENTICATOR_REFRESH_AGE}")
```

Comment:
Might need some alignment with `MAGPIE_COOKIE_EXPIRE`.

Reply:
In case the value is set higher here than the actual cookie age. It would give the false feedback that the cookie/session is still valid until the next refresh happens, while Magpie would actually log the user out. The refresh rate should happen at least a few times per cookie-age/refresh cycle to provide a relevant status.

Reply:
Ok, I understand.

Since the default MAGPIE_COOKIE_EXPIRE value is 24 hours and this checks every minute now. That should be more that sufficient to align the values